sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -107,7 +114,16 @@ def __init__(
         self.average_precision = average_precision
         self.pos_label = pos_label
 
-    def plot(self, ax=None, *, name=None, **kwargs):
+    def plot(
+        self,
+        ax=None,
+        *,
+        name=None,
+        plot_chance_level=False,
+        pos_prevalence=0,
```

Comment:
I think setting a default value `pos_prevalence=0` may lead the user into thinking that the chance level is indeed AP=0.

Instead maybe we can set `pos_prevalence=None` by default and even raise a warning/error that one needs to explicitly pass a value for `pos_prevalence` if `plot_chance_level=True`.